### PR TITLE
DROOLS-5743 : Performing test is slow in Business central

### PR DESCRIPTION
**Pull request by Rikkola**
**JIRA**: 
[link](https://issues.redhat.com/browse/DROOLS-5743)

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

**Referenced Pull Requests**:
* https://github.com/kiegroup/kie-wb-common/pull/3458
* https://github.com/kiegroup/drools-wb/pull/1434

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
 
* <b>a full downstream build</b> please add comment: <b>Jenkins run fdb</b>
  
* <b>a compile downstream build</b> please  add comment: <b>Jenkins run cdb</b>

* <b>a full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>

* <b>an upstream build</b> please add comment: <b>Jenkins run upstream</b>
</details>

### CODE REVIEW

**Comment by yesamer** [comment]
jenkins retest this please

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/test/java/org/drools/workbench/screens/scenariosimulation/backend/server/ScenarioRunnerServiceImplTest.java [hunk 1]
```diff
@@ -70,13 +68,11 @@
     @Mock
```
Comment (yesamer) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/test/java/org/drools/workbench/screens/scenariosimulation/backend/server/ScenarioRunnerServiceImplTest.java:68]: Not related to your PR: `moduleServiceMock` seems not used. If true, I suggest to remove it.

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/test/java/org/drools/workbench/screens/scenariosimulation/backend/server/ScenarioRunnerServiceImplTest.java [hunk 2]
```diff
@@ -141,6 +134,8 @@ public void runTest() throws Exception {
 
     @Test
     public void runTestWithScenarios() throws Exception {
```
Comment (yesamer) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/test/java/org/drools/workbench/screens/scenariosimulation/backend/server/ScenarioRunnerServiceImplTest.java:136]: These `throws Exception` can be removed.

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/main/java/org/drools/workbench/screens/scenariosimulation/backend/server/AbstractKieContainerService.java [hunk 3]
```diff
@@ -35,25 +32,14 @@
...
-            return assertKieContainerIsValid(clone.getKieContainer());
+        KieModule kieModule = moduleService.resolveModule(path);
+        KieContainer kieContainer = buildInfoService.getBuildInfo(kieModule).getKieContainer();
+        if (kieContainer == null) {
+            throw new IllegalArgumentException("Retrieving KieContainer has failed. Fix all compilation errors within the " +
+                                                       "project and build the project again.");
         }
-
-        throw new IllegalStateException("Failed to clone Builder.");
+        return assertKieContainerIsValid(kieContainer);
```
Comment (yesamer) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/main/java/org/drools/workbench/screens/scenariosimulation/backend/server/AbstractKieContainerService.java:41]: @Rikkola `kieContainer` null check is performed 2 times: line 37 and line 41 calling `assertKieContainerIsValid()`. Please remove one of them.

**Review by yesamer — APPROVED** [review]
@Rikkola Approved, the PR correctly reverts the behavior. I just have 3 minor suggestions

**Comment by yesamer** [comment]
jenkins retest this please

**Comment by yesamer** [comment]
@Rikkola @jstastny-cz Considering we need to include it in 7.9.1, I suggest to set it as Ready for Review

**Comment by jstastny-cz** [comment]
jenkins do fdb

**Comment by jstastny-cz** [comment]
jenkins do fdb

**Comment by jstastny-cz** [comment]
going to run full regression on the FDB.

**Comment by Rikkola** [comment]
jenkins retest this please

**Comment by Rikkola** [comment]
Jenkins retest this

**Comment by yesamer** [comment]
Asked @karreiro to review it in order to have 2 eng. approvals

**Review by yesamer — APPROVED** [review]

**Review by karreiro — APPROVED** [review]
Thank you, @Rikkola!

**Review by tomasdavidorg — APPROVED** [review]

**Pull request merged**